File: app/app/Model/Discovery/Response/MemberDiscoveryResponse.swift
struct MemberDiscoveryResponse: Codable, Identifiable {

    let memberId: Int64
    let profileUrl: String?
    let nickname: String
    let gender: String
    let age: Int
    let comment: String?
    let distance: Double?
    let likes: Int
    let updatedAt: String

    var id: Int64 { memberId }
}

File: server/src/main/kotlin/com/pidulgi/server/discovery/dto/response/MemberDiscoveryResponse.kt
package com.pidulgi.server.discovery.dto.response

import com.pidulgi.server.member.entity.type.Gender
import java.time.LocalDateTime

data class MemberDiscoveryResponse(

    val memberId: Long,
    val profileUrl: String?,
    val nickname: String,
    val gender: Gender,
    val age: Int,
    val comment: String?,
    val distance: Double?,
    val likes: Int,
    val updatedAt: LocalDateTime,
)
